packages/camera/camera_platform_interface/lib/src/platform_interface/camera_platform.dart

```diff
@@ -272,6 +272,30 @@ abstract class CameraPlatform extends PlatformInterface {
     throw UnimplementedError('setZoomLevel() is not implemented.');
   }
 
+  /// Gets a list of video stabilization modes that are supported for the selected camera.
+  ///
+  /// Will return the list of supported video stabilization modes
+  /// on Android (when using camera_android_camerax package) and
+  /// on iOS. Will return an empty list on all other platforms.
```

Comment:
Platform interfaces should not make claims about the behavior of specific platforms.